src/main/java/org/jabref/logic/util/strings/QuotedStringTokenizer.java

```diff
@@ -23,9 +23,9 @@ public QuotedStringTokenizer(String content, String delimiters, char quoteCharac
         quoteChar = quoteCharacter;
         contentLength = this.content.length();
         // skip leading delimiters
-        while (isDelimiter(this.content.charAt(index)) && index < contentLength) {
-            ++index;
-        }
+//        while (isDelimiter(this.content.charAt(index)) && index < contentLength) {
```

Comment:
Why is this commented out? Can this be removed?

Reply:
I really have the suspicion, that this is not meant to be commented out, because this would surely have unwanted side effects.